contrib/storage-hbase/src/main/java/org/apache/drill/exec/store/hbase/HBaseGroupScan.java

```diff
@@ -382,7 +402,7 @@ public String getDigest() {
   public String toString() {
     return "HBaseGroupScan [HBaseScanSpec="
```

Comment:
Can we update this method with the `PlanStringBuilder` as shown below?   Please update with appropriate fields
```
  @Override
  public String toString() {
    return new PlanStringBuilder(this)
      .field("HbaseScanSpec", hbaseScanSpec)
      .field("columns", columns)
      .field("maxRecords", maxRecords)
      .toString();
  }
```